docs/Researcher/user-interface/trainings.md

```diff
@@ -28,23 +28,24 @@ To add a training:
 
 1. Press **Tranings** in the menu.
 2. In the *Projects* pane, select the destination project. Use the search box to find projects that are not listed. If you can't find the project, see your system administrator.
-3. In the *Templates* pane, select a template from the list. Use the search box to find templates that are not listed. If you can't find the specific template you need, see your system administrator.
-4. In the *Training name* pane, enter a name for the *Traninng*, then press continue.
-5. In the *Environment* pane select or [create a new environment](workspaces/create/create-env.md). Use the search box to find environments that are not listed.
-6. In the *Compute resource* pane, select resources for your tranings or [create a new compute resource](workspaces/create/create-compute.md). Use the search box to find resources that are not listed. Press *More settings* to use **Node Affinity** to limit the resources to a specific node.
-7. Open the *Volume* pane, and press *Volume* to add a volume to your training.
+3. In the *Multi-node* pane, choose `Single node` for a single node training, or `Multi-node (distributed)` for distributed training. When you choose `Multi-node`, you will then choose a framework that is listed. Then choose the `multi-node` training configuration by selecting either `Workers & master` or `Workers only`.
+4. In the *Templates* pane, select a template from the list. Use the search box to find templates that are not listed. If you can't find the specific template you need, see your system administrator.
```

Comment:
user can also create his own templates ("seeing his system administrator" is not the only way here)

Reply:
we should also state that the template options appear in the temple section derive from the selection made above (project section & multi-node section)

Reply:
Sreps 3-7 were deleted from the procedure. Is there a reason why you deleted it?

Reply:
@jasonnovichRunAI I don't remember deleting anything. In general, I only comment & will never edit myself